.env

```diff
@@ -0,0 +1 @@
+COMPOSE_PROJECT_NAME=hca
```

Comment:
Actually if we use `hca` as the project name and name the individual compose image as `dcp_dss`, we can get to `hca_dcp_dss`.